custom/icds_reports/tasks.py

```diff
@@ -184,6 +186,17 @@ def _aggregate_gm_forms(day):
         AggregateGrowthMonitoringForms.aggregate(state_id, force_to_date(agg_date))
 
 
+@track_time
+def _aggregate_child_health_pnc_forms(day):
+    state_ids = (SQLLocation.objects
+                 .filter(domain=DASHBOARD_DOMAIN, location_type__name='state')
+                 .values_list('location_id', flat=True))
+
+    agg_date = force_to_date(day)
+    for state_id in state_ids:
+        AggregateChildHealthPostnatalCareForms.aggregate(state_id, force_to_date(agg_date))
```

Comment:
nit: `force_to_date()` was called 2 lines up. Is there a reason it gets called twice?